CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+### 1.0.9
+
+* Hide panels by settings on start.
```

Comment:
Can you expand on this?  Something more like "added new option [insert option] to hide panels on start, defaults to false"

Reply:
Hope that's enough 👍🏻